logistical.js

```diff
@@ -172,19 +178,25 @@ Classifier.prototype.train = function(expectedValue, data) {
 
 /*
  * Predicts the probability of the specified value for the given data.
+ *
+ * w = Vector coefficients
+ * data = Matrix of training data vecors (X1, X2, ..., Xn)
  */
 Classifier.prototype.predict = function(w, data) {
   return this.logistic(w.dot(data));
-}
+};
 
 /*
  * Classifies the provided example.
+ *
+ * w = Vector coefficients
+ * data = Matrix of training data vecors (X1, X2, ..., Xn)
  */
 Classifier.prototype.classify = function(w, data) {
   var prediction = this.predict(w, data);
 
   // if the prediction is over 1/2, classify as 1 otherwise 0
-  return (prediction > 0.5);
+  return (prediction > 0.5) ? 1 : 0;
```

Comment:
Really? There is already a comment, I don't know that this is explicitly necessary. I think we should either remove the comment or remove the inline conditional.